CodeForPoznan/alinka-electron
# Partial rendering of INDYWIDUALNE decision

How to test:
* Open `src/docx/index.js` and change values in `generateDocument` execution second parameter.
* `npm install`
* `node src/docs/index.js`
* Open `output.docx`.

There are still some options to be renered based on `data but that should be proper start to refine on.
